### ISSUE TITLE
[10.0] web_responsibe - Attendance Kiosk Mode won't go full page

### ISSUE THREAD

**Issue by Ark74** [OCA/web#859]
Hi!,

Odoo 10 - https://github.com/odoo/odoo/commit/66e76d681d41b6c007a0394b078ddbe872989985
web_responsibe - https://github.com/OCA/web/commit/8a3afd4173f4c7a3e16d11a93bf96229d3a286b7

When installing the attendance app/module, if we launch the Kiosk Mode.
The top header menu bar won't go away, I've looked for and seems like the page should become a full page with no other way to manipulate the site as for a closer behavior of a kiosk mode.

Here my only evidence where I can base this thought.
https://youtu.be/m9Nbv2Xl12k?t=22s

Here the actual result:
![kioskmode](https://user-images.githubusercontent.com/7444518/35959689-3f88d762-0c6c-11e8-9c81-8916f5211151.png)

**Comment by pedrobaeza** [comment]
Yeah, I confirm the problem. Menu bar should go away.

**Comment by Ark74** [comment]
Ohh, so it is how it works, cool!
I thought that there was an special url or something like it, seems to me that needing to log as an Employee Officer to allow all the crew to login could be kind tricky.
How can I help to fix this?

Thanks.

**Comment by pedrobaeza** [comment]
You need to check the JS/CSS trick that is used in standard and apply it here on web_responsive.

**Comment by Ark74** [comment]
Ok, sounds simple enough :smile: , I'll start playing with with see what comes out.
Thanks.

**Comment by Ark74** [comment]
Hi, me again!
The hr kiosk attendance mode launches fullscreen,

```
    <record id="hr_attendance_action_kiosk_mode" model="ir.actions.client">
        <field name="name">Attendances</field>
        <field name="tag">hr_attendance_kiosk_mode</field>
        <field name="target">fullscreen</field>
    </record>
```
I'm not sure but i believe this is what it calls,
https://github.com/odoo/odoo/blob/cd48a806ac636806130efcf5c055f73758cb0902/addons/web/static/src/less/webclient_layout.less#L8

Seems like web_responsive has no fullscreen mode.

I've been testing adding it here,
https://github.com/OCA/web/blob/dafa0a707b1bce0bdeaf25a01cf8a223deabbe06/web_responsive/static/src/less/main.less

With no result. I'll keep digging :smile:

**Comment by pedrobaeza** [comment]
There should be JS code that makes some changes in the DOM when it receives that target. @Yajo do you know about this?

**Comment by Ark74** [comment]
@JayVora-SerpentCS could you post the fix you talked about?

**Comment by JayVora-SerpentCS** [comment]
I will make a PR soon. Thanks

**Comment by yajo** [comment]
No idea :confused:

**Comment by pedrobaeza** [comment]
Fixed in #873

**Comment by mgielissen** [comment]
Can this also be fixed for 11.0?

**Comment by pedrobaeza** [comment]
Feel free to propose the same patch in a new PR for v11.
